projects/plugins/jetpack/modules/sso.php

```diff
@@ -219,8 +219,15 @@ public function render_invitation_email_message() {
 		// Enqueue the CSS for the admin create user page.
 		wp_enqueue_style( 'jetpack-sso-admin-create-user', plugins_url( 'modules/sso/jetpack-sso-admin-create-user.css', JETPACK__PLUGIN_FILE ), array(), time() );
 
+		$message = sprintf(
+			'%s<a class="%s" rel="noopener noreferrer" target="_blank" href="%s">%s</a>',
+			__( 'New users will receive an invite to join WordPress.com, so they can log in securely using', 'jetpack' ),
+			'jetpack-sso-admin-create-user-invite-message-link-sso',
+			esc_url( 'https://jetpack.com/support/sso/' ),
+			__( 'Secure Sign On.', 'jetpack' )
+		);
```

Comment:
This is working great, a few notes:
- Let's use %1$s, %2$s, %3$s instead, to have more clarity.
- After that let's add the `* translators: %1$s is the message, %2$2 is ....`